libnavigation-util/src/main/java/com/mapbox/navigation/utils/thread/WorkThreadHandler.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.navigation.utils.thread
+
+import android.os.Handler
+import android.os.HandlerThread
+import com.mapbox.navigation.utils.extensions.quitSafelySupport
+import timber.log.Timber
+
+class WorkThreadHandler(
```

Comment:
Same as above re: `internal` and `internal` package.

Reply:
It is inside utils module. It should not be internal. We should be able to use it across SDK by adding proper module to dependencies, as it working now. WDYT?

Reply:
Same as above ☝️